src/main/java/ua/lviv/iot/parkingServer/kafka/EventObserver.kt

```diff
@@ -0,0 +1,22 @@
+package ua.lviv.iot.parkingServer.kafka
+
+import io.grpc.ManagedChannel
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import ua.lviv.iot.ParkingSpotOuterClass.ParkingSpotRequest
+import ua.lviv.iot.ReactorKafkaParkingSpotServiceGrpc
+
+@Component
+class EventObserver(private val grpcChannel: ManagedChannel) {
```

Comment:
why do you need an observer here? Looks like some class for tests. if so, move it to test module, please

Reply:
I'm not sure, because this class responsible for observing parking spot events via gRPC